### DISCUSSION TITLE
Nuxt 2 or Nuxt 3 for a long new project

### DISCUSSION THREAD

**Discussion by narduin** [nuxt/nuxt#17616] [General]
Hello!

I've been using Nuxt 2 for several years making static websites.
I'm about to start a new website project (couple weeks maybe).
The dev time of this project is estimated at 6/7 months. So production is supposed to happen around december 2022/january 2023.

I've been experimenting with Nuxt 3 this past week and while it's already robust and stable, **I was wondering if it was worth it to start this new project with Nuxt 3.**

My thoughts are as follows:

- Nuxt 2 is stable and has all the features I need (`nuxt-image` and `nuxt generate` essentially)
- But in 10 to 12 months it might not be relevant to still work on/maintain Nuxt 2
- `nuxt-image`and `nuxt generate` are works in progress for Nuxt 3
- I'm confident they will be available before the end of the project but could be released towards the end of the year which could cause problems or delays during development

This is a weird time to start new projects with nuxt 😄 
What are your thoughts on this?

**Comment by CodyBontecou** [comment]
I'd create the project using Nuxt 2 alongside Nuxt Bridge and build the application using Vue 3 and the Composition API. This will give you access to the latest Vue features and allow for easier migration to Nuxt 3 in the future.

**Reply by narduin** [reply]
Alright good point. I was not sure a full migration to Nuxt 3 would be possible when ready.
Thanks!

**Reply by P4sca1** [reply]
Nuxt Bridge has its rough edges (e.g. https://github.com/nuxt/bridge/issues/344) and also does not support every module that Nuxt 2 supports.
You can check here if you require any module that is not supported by bridge: https://modules.nuxtjs.org
